source/simulator/core.cc

```diff
@@ -2246,9 +2246,36 @@ namespace aspect
     if (nonlinear_solver_failures > 0)
       pcout << "\nWARNING: During this computation " << nonlinear_solver_failures << " nonlinear solver failures occurred!" << std::endl;
 
-    pcout << "-- Total wallclock time elapsed including restarts: "
-          << std::round(wall_timer.wall_time()+total_walltime_until_last_snapshot)
-          << 's' << std::endl;
+    std::ostringstream resource_output;
+    resource_output << std::setprecision(2);
+
+    const double wallclock_time = wall_timer.wall_time()+total_walltime_until_last_snapshot;
+    const double resource_usage = wallclock_time / 3600. * Utilities::MPI::n_mpi_processes(mpi_communicator);
```

Comment:
It's perhaps not worth worrying about, but at least conceptually: We support restarts with a different number of processes. Addressing this would likely be more effort than is worth it.